dask_expr/_collection.py

```diff
@@ -4638,9 +4642,59 @@ def read_parquet(
             col, op, val = filter
             if op == "in" and not isinstance(val, (set, list, tuple)):
                 raise TypeError("Value of 'in' filter must be a list, set or tuple.")
+    from pyarrow import fs as pa_fs
+
+    if (
+        isinstance(filesystem, pa_fs.FileSystem)
```

Comment:
do you want to make those configs raise or should we rather fall back to the old implementation?

Reply:
The old implementation is currently pretty broken so I don't want to fall back, see https://github.com/dask-contrib/dask-expr/issues/856